torch/ao/quantization/experimental/fake_quantize_helper.py

```diff
@@ -0,0 +1,28 @@
+import torch
```

Comment:
what is the purpose of this file? seems to be a duplicate of `fake_quantize_fucntion.py`?

Reply:
yep removing this